packages/terra-overlay/src/Overlay.jsx

```diff
@@ -12,7 +12,9 @@ import Container from './OverlayContainer';
 // Importing WICG Inert polyfill causes Jest to crash
 // Issue logged to Jest repo: https://github.com/facebook/jest/issues/8373
 // This logic avoids importing the polyfill when running Jest tests
-if (process.env.NODE_ENV !== 'test') {
+// It also checks to make sure not to load the inert polyfill if it has already been defined
+// eslint-disable-next-line no-prototype-builtins
+if (process.env.NODE_ENV !== 'test' && !Element.prototype.hasOwnProperty('inert')) {
   // eslint-disable-next-line global-require
   require('wicg-inert');
```

Comment:
Is this the only place we require this?  (i.e. abstract modal)

Reply:
This and abstract modal are the only places we use the inert polyfill. Planning to merge this down and then add the same change in abstract modal.